install.sh

```diff
@@ -455,53 +457,71 @@ __gather_pkg_mgr() {
 	fi
 }
 
-
-__gather_bro() {
-	_BRO_PATH=""
-	_BRO_PKG_INSTALLED=false
+__bro_installed() {
+	_BRO_INSTALLED=false
 	if __package_installed bro; then
-		_BRO_PKG_INSTALLED=true
-		_BRO_PATH="/opt/bro/bin"
+		_BRO_INSTALLED=true
+	elif [ -f "/usr/local/bro/bin/bro" ]; then
+		_BRO_INSTALLED=true
 	fi
 
-	_BRO_ONION_INSTALLED=false
+	if [ "$_BRO_INSTALLED" = "true" ]; then
+		printf "It looks like Bro is installed on this system. This version of RITA uses Zeek.\n"
+		printf "For the best results, please stop the script, uninstall Bro, and re-run the script.\n"
```

Comment:
We should consider at least writing up a document with manual steps needed to uninstall Bro in preparation for Zeek.

Reply:
On that note, how do you feel about also replacing/removing [Bro Install From Source.md](https://github.com/activecm/rita/blob/master/docs/Bro%20Install%20From%20Source.md)?

Reply:
Removing it is good with me.

Reply:
Done.